packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -67,134 +77,138 @@ const BlockActionsMenu = ( {
 		},
 	} = getMoversSetup( isStackedHorizontally, moversOptions );
 
-	const deleteOption = {
```

Comment:
NOTE: This looks like a huge change, but I'm actually just refactoring to one object instead of having multiple objects, one for each option. This makes it easier to read the code and allows code folding for better readability 🎉 .